pkg/security/resolvers/resolvers_ebpfless.go

```diff
@@ -47,7 +51,15 @@ func NewEBPFLessResolvers(config *config.Config, statsdClient statsd.ClientInter
 		return nil, err
 	}
 
-	cgroupsResolver, err := cgroup.NewResolver(tagsResolver)
+	var sbomResolver *sbom.Resolver
+	if config.RuntimeSecurity.SBOMResolverEnabled {
+		sbomResolver, err = sbom.NewSBOMResolver(config.RuntimeSecurity, statsdClient, wmeta)
+		if err != nil {
+			return nil, err
+		}
+	}
+
+	cgroupsResolver, err := cgroup.NewResolver(tagsResolver, sbomResolver)
```

Comment:
As we don't have yet dump/profiles for ebpfless, I wonder what will be the impact of having the SBOM resolver initialized here.